recipes/qt/6.x.x/conanfile.py

```diff
@@ -297,38 +297,38 @@ def layout(self):
     def requirements(self):
         self.requires("zlib/1.2.13")
         if self.options.openssl:
-            self.requires("openssl/1.1.1t")
+            self.requires("openssl/[>=1.1 <4]")
```

Comment:
Its my understanding that Qt6 does not support openssl 3.x yet.  There is a [blog post](https://www.qt.io/blog/moving-to-openssl-3-in-binary-builds-starting-from-qt-6.5-beta-2) announcing the intent to add support but I did not see it in the 6.5.0 or 6.5.1 release notes

Reply:
This blog post says qt6 supports openssl 3 since qt 6.2.0. The migration mentioned in this blog post is just in the environment used to generate official qt binaries

Reply:
Ah. I misread that. Thanks!